lib/api.js

```diff
@@ -97,14 +107,14 @@ const processRequest = function(req, res) {
                     warnings2 = [];
                     info2 = [];
                     v2 = new validator.Specberus();
-                    handler2 = new Sink(function(...data2) {
-                        errors2.push(Object.assign({}, ...data2));
+                    handler2 = new Sink(function (...data2) {
```

Comment:
This is not related to your changes but `handler2` should also listen to `exception` events like `handle` does on l.123